spec/features/female_missing_info_feature_spec.rb

```diff
@@ -119,4 +126,39 @@
       wait_for { page.current_path == missing_information_prison_prisoners_path(prison.code) }
     end
   end
+
+  context 'when multiple tabs are opened', :js do
+    let(:complexity) { 'medium' }
+
+    it 'can complete forms in parallel' do
+      visit missing_information_prison_prisoners_path prison.code
+      window = page.open_new_window
+      current = page.current_window
```

Comment:
These variable names are a bit unclear.

Would it be possible to name them `tab_1` and `tab_2` or something – to make it really obvious that we're switching between 2 tabs in this test?